src/components/ContentPreview/ContentPreview.js

```diff
@@ -425,22 +437,37 @@ class ContentPreview extends PureComponent<Props, State> {
     }
 
     /**
-     * Event handler 'preview_metric' which adds in the file fetch time
+     * Event handler 'preview_metric' which also adds in the file fetch time if it's a load event
      *
-     * @param {Object} previewLoadMetrics - the object emitted by 'preview_metric'
+     * @param {Object} previewMetrics - the object emitted by 'preview_metric'
      * @return {void}
      */
-    onPreviewMetric = (previewLoadMetrics: PreviewLoadMetrics): void => {
+    onPreviewMetric = (previewMetrics: PreviewMetrics): void => {
         const { onMetric }: Props = this.props;
-        const totalFetchFileTime = this.getTotalFileFetchTime();
+        const { event_name } = previewMetrics;
+        let metrics = {
+            ...previewMetrics
+        };
 
         // We need to add in the total file fetch time to the file_info_time and value (total)
         // as preview does not do the files call
-        onMetric({
-            ...previewLoadMetrics,
-            file_info_time: totalFetchFileTime,
-            value: (previewLoadMetrics.value || 0) + totalFetchFileTime
-        });
+        if (event_name === 'load') {
```

Comment:
Would this make sense as a constant? It's probably somewhere else in Elements already